scripts/electron_afterSign.js

```diff
@@ -0,0 +1,26 @@
+const { notarize } = require('electron-notarize');
+
+exports.default = async function(context) {
+    const { electronPlatformName, appOutDir } = context;
+    if (electronPlatformName !== 'darwin') {
+        return;
+    }
+
+    // We get the password from keychain. The keychain stores
+    // user IDs too, but apparently altool can't get the user ID
+    // from the keychain, so we need to get it from the environment.
+    const userId = process.env.NOTARIZE_APPLE_ID;
```

Comment:
Who will set this env var? Can we manually extract it from keychain with `keychain find-generic-password ...` or similar?

Reply:
I was just going to put it in the ~/.profile on the build machine. I'd hoped we'd be be able to just get it out of the keychain but I can't work out how to get the entry in a place in the keychain where both `altool` and `security` can find it...

Reply:
Aha, I see... Well, probably it's fine. 😄

Reply:
What's the current method for getting this set up?  I've tried `NOTARIZE_CREDS=foo yarn build:electron:macos` to no avail...

Reply:
Let's discuss in this #riot-dev.